PowerUp/app/src/main/java/powerup/systers/com/DisplayPointsActivity.java

```diff
@@ -0,0 +1,56 @@
+package powerup.systers.com;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import powerup.systers.com.datamodel.SessionHistory;
+
+
+/**
+ * Created by chhavi on 9/1/16.
+ */
+public class DisplayPointsActivity extends Activity {
+    private TextView currentPoints;
```

Comment:
Give better names to variables making obvious what they are. It should be clear this is a textview and not a string or an integer from its name `currentPointsTextView` is a better name, for example